### PR TITLE
UHM-5316 Migrate "Note" encounter type

### PR DESCRIPTION
**Pull request by brandones**
https://pihemr.atlassian.net/browse/UHM-5316

### CODE REVIEW

**Review by mogoodrich — COMMENTED** [review]
One question...

**Comment by brandones** [comment]
@mogoodrich Yes?

File: etl/src/main/java/org/pih/hivmigration/etl/sql/EncounterMigrator.groovy [hunk 1]
```diff
@@ -66,23 +71,17 @@ class EncounterMigrator extends SqlMigrator {
...
-            
+        executeMysql("Fill encounter types column", '''            
             UPDATE hivmigration_encounters SET encounter_type_id = CASE
-                WHEN source_encounter_type = "intake" THEN @encounter_type_intake
-                WHEN source_encounter_type = "followup" THEN @encounter_type_followup
-                WHEN source_encounter_type = "lab_result" THEN @encounter_type_lab_results
-                WHEN source_encounter_type = "anlap_lab_result" THEN @encounter_type_lab_results
-                WHEN source_encounter_type = "accompagnateur" THEN @encounter_type_drug_dispensing
-                WHEN source_encounter_type = "regime" THEN @encounter_type_drug_order
+                WHEN source_encounter_type = "intake" THEN encounter_type('HIV Intake')
```
Comment (mogoodrich) [inline etl/src/main/java/org/pih/hivmigration/etl/sql/EncounterMigrator.groovy:76]: Where is this encounter_type function defined?  Does this come from something we use for reporting?

**Review by mogoodrich — COMMENTED** [review]
Whoops @brandones ... not sure what happened to my previous review but I added my question again... :)

File: etl/src/main/java/org/pih/hivmigration/etl/sql/EncounterMigrator.groovy (hunk 1 above)
Comment (brandones) [reply etl/src/main/java/org/pih/hivmigration/etl/sql/EncounterMigrator.groovy:76]: I think it must. @dmdesimone ?

**Review by mogoodrich — APPROVED** [review]
Okay, LGTM!

**Pull request merged**